entries/s/solarwinds.com.json

```
@@ -0,0 +1,18 @@
+{
+  "SolarWinds": {
+    "domain": "solarwinds.com",
+    "additional-domains": [
+      "pingdom.com",
+      "papertrail.com"
```

Two more related domains, and "solarwinds.cloud" for the login service.
```suggestion
      "papertrail.com",
      "appoptics.com",
      "loggly.com",
      "solarwinds.cloud"
```